lib/matplotlib/backends/backend_qt5.py

```diff
@@ -555,23 +555,21 @@ def __init__(self, canvas, num):
             if self.toolbar:
                 backend_tools.add_tools_to_container(self.toolbar)
                 self.statusbar = StatusbarQt(self.window, self.toolmanager)
+                sb_height = self.statusbar.sizeHint().height()
+        else:
+            sbs_height = 0
```

Comment:
Typo? Is this the same as `sb_height`?

Reply:
actually they should all be sbs_height, fixed